usaspending_api/api_contracts/contracts/v2/search/spending_by_award.md

```diff
@@ -172,6 +172,8 @@ List of table columns
 + `set_aside_type_codes`: `NONE` (optional, array[string])
```

Comment:
Same updates to usaspending_api/api_contracts/contracts/v2/search/spending_by_award_count.md?